bundles/org.openhab.binding.hue/src/main/resources/ESH-INF/thing/HumiditySensor.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Humidity Sensor -->
+	<thing-type id="0405">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Hue Humidity Sensor</label>
+		<description>A sensor providing relative humidity values.</description>
+
+		<channels>
+			<channel id="humidity" typeId="humidity" />
+			<channel id="last_updated" typeId="last_updated" />
+			<channel id="battery_level" typeId="system.battery-level" />
+			<channel id="battery_low" typeId="system.low-battery" />
+		</channels>
+
+		<representation-property>uniqueId</representation-property>
+
+		<config-description>
```

Comment:
I have introduced the config-description, for all things except PresenceSensor (0107) and LightLevelSensor (0106) because they have special parameters. Please have a look.